hack/jenkins/e2e.sh

```diff
@@ -93,6 +97,22 @@ GCE_DEFAULT_SKIP_TESTS=(
     "Example"
     )
 
+# Tests which cannot be run on GKE, e.g. because they require
+# master ssh access.
+GKE_REQUIRED_SKIP_TESTS=(
+    "Nodes"
+    "Etcd\sFailure"
+    "MasterCerts"
+    "Shell"
+    )
+
+GKE_REBOOT_SKIP_TESTS=(
```

Comment:
Oh, wait, I see that you don't currently skip GCE_DEFAULT_SKIP_TESTS in your reboot test.  Never mind.  You could potentially use --ginkgo-focus="Reboot" instead, but I don't think that's what you're trying to do here.  You're trying to run all the same tests as e2e-gce, and in addition "Reboot", right?  We should perhaps refactor all of this a bit to remove the duplication between GCE_DEFAULT_SKIP_TESTS and  GKE_REBOOT_SKIP_TESTS, as it seems that they always need to be identical with the exception of "Reboot"?